internal/contact/email.go

```diff
@@ -32,17 +32,17 @@ func NewContactService(emailRepository EmailRepository, configManager ConfigMana
 	return &contact{emailRepository: emailRepository, configManager: configManager, stage: stage}
 }
 
-func (e *contact) SendEmail(ctx context.Context, data models.EmailContactRequest) error {
+func (e *contact) SendEmail(ctx context.Context, req models.EmailContactRequest) error {
```

Comment:
I'm not sure if I agree with this renaming, but I suppose that's just a matter of preference.

Reply:
oh. I guess I will change this back.